internal/support/scheduler/application/intervalaction.go

```diff
@@ -106,6 +106,9 @@ func PatchIntervalAction(dto dtos.UpdateIntervalAction, ctx context.Context, dic
 	var action models.IntervalAction
 	var edgeXerr errors.EdgeX
 	if dto.Id != nil {
+		if *dto.Id == "" {
+			return errors.NewCommonEdgeX(errors.KindContractInvalid, "id is empty", nil)
```

Comment:
in this case, it should get by name
after validation, name should contain value.

Reply:
Got it, we can get by name if the id is nil or empty string.